assets/src/edit-story/components/richText/karma/inlineSelection.karma.js

```diff
@@ -358,6 +360,7 @@ describe('CUJ: Creator can Add and Write Text: Select an individual word to edit
       // Change line height to 5
       await data.fixture.events.click(lineHeight, { clickCount: 3 });
       await data.fixture.events.keyboard.type('5');
+      await data.fixture.events.keyboard.press('Enter');
       await data.fixture.events.keyboard.press('Escape');
```

Comment:
In the tests, was the `Escape` previously for submitting the changes? Could these be removed now, from here and elsewhere?